src/dotnet/APIView/APIViewWeb/Client/tests/unit/shared/helpers.spec.ts

```diff
@@ -9,4 +9,93 @@ test('getCookies return valid single cookie', async ({ page }) => {
   expect(hp.getCookieValue(testCookieString, 'tz')).toBe('America%2FLos_Angeles');
   expect(hp.getCookieValue(testCookieString, '_octo')).toBe('GH1.1.1383888135.1683830525');
   expect(hp.getCookieValue(testCookieString, 'invalidCookie')).toBe(null);
-});
\ No newline at end of file
+});
+
+test.describe('getReviewAndRevisionIdFromUrl should return valid review and revision id on reviews, null otherwise', () => {
```

Comment:
Looks good. I think you can probably tidy up the code and avoid repetitions by [parameterizing the test](https://playwright.dev/docs/test-parameterize#parameterized-tests). If you add all the test setup and expected result in a data structure should reduce the number of lines here.